src/sql/platform/credentials/common/credentialsService.ts

```diff
@@ -8,7 +8,7 @@ import { createDecorator } from 'vs/platform/instantiation/common/instantiation'
 import * as azdata from 'azdata';
 import { Deferred } from 'sql/base/common/promise';
 
-export const SERVICE_ID = 'credentialsService';
+export const SERVICE_ID = 'sqlCredentialsService';
```

Comment:
Had to change this because it clashes with VSCode's singleton `credentialService` decorator which is required to use the native credential service